projects/elife.yaml

```diff
@@ -501,6 +501,13 @@ journal:
                 engine: redis
                 type: cache.t2.micro
                 clusters: 2
+            elb:
+                stickiness:
```

Comment:
we probably don't need this anymore, but identical to the existing configurations

Reply:
We don't need this anymore, and I think it's broken anyway as the neither CDN whitelists the `AWSELB` cookie.

Reply:
Will remove as a follow up.